### ISSUE TITLE
Instructor's view inaccuracies

### ISSUE THREAD

**Issue by viktorradnai** [viktorradnai/flightgear-ask21#79]
The instructor's viewpoint has a few issues. First, the arrangement of the front seat's back and the pedals that go around it look wrong. Please compare the following two pics:

![ask21_instructor_view](https://cloud.githubusercontent.com/assets/4738178/23592949/499f703e-0200-11e7-921b-7ba8decd558b.jpg)

![fgfs-screen-235](https://cloud.githubusercontent.com/assets/4738178/23592967/7969c1d4-0200-11e7-8e9b-e39fabd8e57e.png)

The instructor's stick should also have a button for the radio. The entire footwell should be much more in the shade, I had to brighten the above picture to reveal those details, they are normally lost in the shadows. And there should be a trim indicator on the right armrest, same as the front seat.

**Comment by viktorradnai** [comment]
Ah yes, and the middle of the instrument panel comes down a little, obstructing more of the seat back.

**Comment by viktorradnai** [comment]
I'd like to rearrange the instruments on the panel based on this picture at some point. @gilbertohasnofb and @D-ECHO , will this interfere with your uvmap work?

**Comment by gilbertohasnofb** [comment]
@viktorradnai No, I don't believe changing the instrument position will interfere with the texture and uv-map work in the panels

**Comment by viktorradnai** [comment]
@gilbertohasnofb thanks. I'll do a PR once it's ready.

**Comment by gilbertohasnofb** [comment]
:+1:

**Comment by viktorradnai** [comment]
Found something else that is missing from the model -- the canopy latch rods. These go from each the latch handle into the base of the U shaped fixed canopy frame loop. The same rods are present in the front cockpit as well, they go backwards from the latch handle.

**Comment by viktorradnai** [comment]
![fgfs-screen-310](https://cloud.githubusercontent.com/assets/4738178/24432971/9e919660-141d-11e7-981f-b4947f552344.png)

Looks a lot better now, but everything forward of the pedals should be raw fibreglass. Not sure about the back of the front seat, that might be unpainted as well. Perhaps leave it grey for contrast?

**Comment by viktorradnai** [comment]
Also is it just me or did the pilot figures get bigger?

**Comment by viktorradnai** [comment]
@D-ECHO OK, with different light I can see that the seat has the wrong shape still. Consider that the entire seat (with the armrests) is a single curved sheet of fibreglass. The seat pan is much more curved, like the inside of a bathtub. When looked at from behind, it's possible to see right under it. In your model the armrests are too close to the seatback / seat base and the instructor's feet would jam against them.

![fgfs-screen-311](https://cloud.githubusercontent.com/assets/4738178/24433488/02c8ab48-1421-11e7-8232-cd094128122f.png)

**Comment by gilbertohasnofb** [comment]
> Not sure about the back of the front seat, that might be unpainted as well. Perhaps leave it grey for contrast?

I can make a texture for that.

> Also is it just me or did the pilot figures get bigger?

I don't know what you mean, but I absolutely hate all these human figures in FG, the models are just terrible so I always disable them.

**Comment by gilbertohasnofb** [comment]
@viktorradnai @D-ECHO I have pushed a new version of `parts.png` to the `newint` branch and there you will find the new texture for the back side of the seats:

![seat](https://cloud.githubusercontent.com/assets/5700795/24451508/677c87fe-147f-11e7-8bc7-7679eb57af4d.jpg)

**Comment by viktorradnai** [comment]
@gilbertohasnofb Thanks, is that texture applied though? I haven't seen any difference. Btw, if I wanted to play with textures, what would be the best way to change which one a particular 3d object uses (and which area, of course)? How do you do it?

**Comment by gilbertohasnofb** [comment]
> is that texture applied though?

No, I just pushed the altered texture file, @D-ECHO has to UV-map it to the model.

> Btw, if I wanted to play with textures, what would be the best way to change which one a particular 3d object uses (and which area, of course)?

This is what is called UV-mapping (i.e. applying a texture to a 3D object) and you have to do it in Blender. I am not familiar with it to be honest, which is why I create textures but ask D-ECHO to apply them. Is there something specific that needs texture work?

**Comment by viktorradnai** [comment]
Not really, I just thought that sometimes it might be quicker if i made a change than opening an issue and describing exactly and precisely what I'd like done :) At some point I'd like to learn Blender, currently I'm happy if I can import a model and navigate around it :)

**Comment by gilbertohasnofb** [comment]
@D-ECHO I see that the shape of the instructor's panel has already been fixed, is there anything else to be done with this issue?

**Comment by ghost** [comment]
The pedals, but not short-term

**Comment by viktorradnai** [comment]
I have found the reason for one of the discrepancies that has been bugging me. If you look at the IRL photo, you can see the top of the instructor's instrument panel, despite the view point not being very high -- eg. you can't see the front instrument panel over my shoulder. With the sim, I couldn't replicate this view point. This is because in our model, the top of the instrument panel slopes forward, but in reality it's sloping backwards by about the same angle. See here: https://www.jetphotos.com/photo/8665365

![image](https://user-images.githubusercontent.com/4738178/34446201-f6d059b0-ecd0-11e7-9238-02b9ef019918.png)
![fgfs-screen-353](https://user-images.githubusercontent.com/4738178/34446208-ffa74ec2-ecd0-11e7-813e-cc2d57ac0bb7.png)

Also I think the instrument panel should be black.
